doc/devdocs/settingsv2/project-overview.md

```diff
@@ -0,0 +1,17 @@
+# Overview
+`Settingsv2` is WPF .net core desktop application. It uses the `WindowsXamlHost` control from the Windows Community Toolkit to host UWP controls from `WinUI3` library. More details about WinUI can be found [here](https://microsoft.github.io/microsoft-ui-xaml/about.html#what-is-it).
+
+## Settings V2 Project structure
+The Settings project is a XAML island based project which
```

Comment:
We should create a separate file to call out some of the XAML Island specific workarounds we had to add (we can link to that file here), which should be removed on migrating to WinUI 3, namely:
- Workaround to hide the XAML Island blank icon in the taskbar when the XAML Island application is loading:https://github.com/microsoft/PowerToys/blob/36dd29c05631a54b7c327f52b9bcbcb9ae954e5c/src/core/Microsoft.PowerToys.Settings.UI/App.xaml.cs#L16-L18
- Workaround to prevent XAML Island failing to render on nVidia workstation graphics cards (link to the issue is in the code, but we could link to it in the docs as well)
https://github.com/microsoft/PowerToys/blob/36dd29c05631a54b7c327f52b9bcbcb9ae954e5c/src/core/Microsoft.PowerToys.Settings.UI.Runner/MainWindow.xaml.cs#L83-L87
-  Workaround to ensure XAML Island application terminates if attempted to close from taskbar while minimized.
https://github.com/microsoft/PowerToys/blob/36dd29c05631a54b7c327f52b9bcbcb9ae954e5c/src/core/Microsoft.PowerToys.Settings.UI.Runner/MainWindow.xaml.cs#L90-L101